pkg/cmd/openshift-apiserver/openshiftapiserver/project.go

```diff
@@ -14,17 +14,17 @@ import (
 	quotainformer "github.com/openshift/origin/pkg/quota/generated/informers/internalversion/quota/internalversion"
 )
 
-func NewClusterQuotaMappingController(nsInternalInformer kinternalinformers.NamespaceInformer, cqInternalInformer quotainformer.ClusterResourceQuotaInformer) *clusterquotamapping.ClusterQuotaMappingController {
-	return clusterquotamapping.NewClusterQuotaMappingControllerInternal(nsInternalInformer, cqInternalInformer)
```

Comment:
this is now only used by a test right?  Fix the test.

Reply:
Thanks, done